### PR TITLE
Flux oci support 11: add integration test for  Unable to use Flux OCI with Harbor robot accounts #5219 

### PR DESCRIPTION
**Pull request by gfichtenholt**

### CODE REVIEW

File: cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/gcloud-setup.md [hunk 1]
```diff
@@ -8,20 +8,15 @@ The following service account was set up for use by the integration tests using
...
-  Probably some combination of these and others:
     - Artifact Registry Administrator
     - Artifact Registry Repository Administrator
-    - Artifact Registry Service Agent
+    - Viewer
 
-  Make sure you see a message "Policy Updated" at the bottom of the screen when you grant these roles.
-  If you see "Failed to add project roles" or some other error message,
+  Make sure you see a message "Policy Updated" at the bottom of the screen when you grant these roles. If you see "Failed to add project roles" or some other error message,
   create the service account with a different Name/ID
```
Comment (castelblanque) [inline cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/gcloud-setup.md:20]: Just out of curiosity: is this why `flux-plugin-test-sa-*` is increasing the number?
Do you know the problem behind?

**Review by castelblanque — APPROVED** [review]
LGTM

File: cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/gcloud-setup.md (hunk 1 above)
Comment (gfichtenholt) [reply cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/gcloud-setup.md:20]: No technical reason. There are times when I have had multiple service accounts active for testing, so I increment the Name/ID to keep track of what's what. But I always make sure to clean it up and the end so there is a single service account left
Screen Shot 2022-08-31 at 9 47 51 AM

**Comment by gfichtenholt** [comment]
> LGTM

Thank you

**Pull request merged**